pkg/monitortests/testframework/disruptionexternalcloudservicemonitoring/cloudmonitortest.go

```diff
@@ -0,0 +1,90 @@
+package disruptionexternalcloudservicemonitoring
```

Comment:
Is this module going to be used for gcp or all clouds? It looks setup for just one. If we're doing one package per cloud, then this should be renamed to be specific to gcp throughout.